examples/main.tf

```diff
@@ -4,6 +4,10 @@ provider "aws" {
 }
 
 module "trivy-operator" {
-  source        = "../"
+  source = "../"
```

Comment:
Can we make the example more explicit, i.e. when I copy this example I just want to change the release